Amcovad/amcovad-frontend
# #180822207 Create Tailwind CSS

#### What does this PR do?
This PR creates the setup for tailwind CSS

#### Description of Task to be completed?
- setup tailwind css
- setup jsconfig

#### How should this be manually tested?
- Clone the  repository and run `npm install`
- Run `npm run dev`
- Visit `localhost:3000`
- You should see `Welcome To Amcovad`
